### PR TITLE
Collect start-end position of value in found credential

### PR DESCRIPTION
**Pull request by babenek**
## Description

Please include a summary of the change and which is fixed.

- Add start-end position of found values in line_data_list
- replace tuple of separator position to two separated integers to avoid possible misunderstandings during export to json (tuple is represented as list)

## How has this been tested?

Please describe the tests that you ran to verify your changes.

- [x] UnitTest
- [x] Benchmark

### CODE REVIEW

File: credsweeper/credentials/line_data.py [hunk 1]
```diff
@@ -49,6 +49,9 @@ def __init__(
         self.pattern: re.Pattern = pattern
         # do not store match object due it cannot be pickled with multiprocessing
 
+        # start - end position of matched object
+        self.value_start = -2
```
Comment (babenek) [inline credsweeper/credentials/line_data.py:53]: -2 has been chosen to distinguish with standard -1 value when a string was not found.
Anyway, the value is sub-zero and may be skipped if appears.

File: tests/data/output.json [hunk 2]
```diff
@@ -1762,6 +1926,8 @@
                 "path": "tests/samples/pem_key",
                 "info": "",
                 "value": null,
+                "value_start": -2,
```
Comment (babenek) [inline tests/data/output.json:1929]: The case, when a pattern was not found

File: credsweeper/credentials/line_data.py (hunk 1 above)
Comment (csh519) [reply credsweeper/credentials/line_data.py]: Does -1 has other mean except string was not found?

Comment (csh519) [reply credsweeper/credentials/line_data.py]: I see.

**Review by csh519 — APPROVED** [review]
`value_start` and `value_end` fields are added.

LGTM :+1:

**Pull request merged**